Comanche055/PINBALL_GAME_BUTTONS_AND_LIGHTS.agc

```diff
@@ -2187,7 +2187,7 @@ HMSIN		TC	ALL3DEC		# IF ALL 3 WORDS WERE NOT LOADED, ALARM.
 		MP	MINCON		# LEAVES MINUTES CONTRIBUTION IN A,L
 # Page 353		
 		DAS	HITEMIN		# ADD IN MINUTES CONTRIBUTION
-		EXTEND			# IF THIS DAS OVEFLOWS, G/ 745 HR, 39MIN
```

Comment:
This section of comments seem to be off by one line, but I will allow it. ([source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0353.jpg))